smac/tae/execute_ta_run.py

```diff
@@ -113,7 +118,7 @@ def start(self, config, instance,
         if self.stats.is_budget_exhausted():
             self.logger.debug(
                 "Skip target algorithm run due to exhausted configuration budget")
```

Comment:
I think this should be the message of the exception rather than a call to the logger.